spec/io_resource_spec.rb

```diff
@@ -60,20 +62,20 @@
     context 'IO object' do
       let(:answer) { 'read' }
       let(:rio) { StringIO.new(answer, 'r') }
-      let(:wio) { StringIO.new('write', 'w') }
+      let(:wio) { StringIO.new(+'write', 'w') }
```

Comment:
this is the only noticeable change in this PR. I've searched for potential issues like `<< "` or `= ''` but I couldn't find them